static/app/views/performance/traceDetails/content.tsx

```diff
@@ -422,6 +426,170 @@ class TraceDetailsContent extends Component<Props, State> {
   }
 }
 
+type OnlyOrphanErrorWarningsProps = {
+  organization: Organization;
+  orphanErrors: TraceError[];
+};
+function OnlyOrphanErrorWarnings({orphanErrors}: OnlyOrphanErrorWarningsProps) {
+  const {projects} = useProjects();
+  const projectSlug = orphanErrors[0] ? orphanErrors[0].project_slug : '';
+  const project = projects.find(p => p.slug === projectSlug);
+  const LOCAL_STORAGE_KEY = `${project?.id}:issue-details-replay-onboarding-hide`;
```

Comment:
Should this be using the replay onboarding key? Feels like we'd have a more specific one here